packages/user-group-service/src/groups/typedef.graphql

```diff
@@ -36,25 +32,25 @@ scalar DateTime
 
 type Group {
   _id: ID
+  cn: String
   name: String
-  ldapCommonName: String
   createdOn: DateTime
   updatedOn: DateTime
   members: [String]
 }
 
 input AddGroupInput {
+  cn: String!
   name: String!
-  ldapCommonName: String!
 }
 
 input GetGroupInput {
   _id: ID
-  name: String
-  ldapCommonName: String
+  cn: String!
+  name: String!
```

Comment:
`!` will make both the fields required. It should be optional, where either of the inputs should work.

You can add validation in the resolver to check if at least one is provided.

```suggestion
  cn: String
  name: String
```